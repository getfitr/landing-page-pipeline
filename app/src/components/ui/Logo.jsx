import React from 'react';
import { Box, Text } from "@chakra-ui/layout";

const Logo = (props) => (
  <Box {...props}>
    <Text fontSize="lg" fontWeight="bold">
      GetFitr!
    </Text>
  </Box>
);

export default Logo;
