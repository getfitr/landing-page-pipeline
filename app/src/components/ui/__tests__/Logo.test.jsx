import React from 'react';
import renderer from 'react-test-renderer';
import Logo from '../Logo';

test('Displays the Logo', () => {
  const logo = renderer.create(<Logo />)
  const tree = logo.toJSON()
  expect(tree).toMatchSnapshot();
});