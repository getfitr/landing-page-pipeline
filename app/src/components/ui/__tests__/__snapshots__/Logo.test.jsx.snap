// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays the Logo 1`] = `
<div
  className="css-0"
>
  <p
    className="chakra-text css-1hgmzub"
  >
    GetFitr!
  </p>
</div>
`;
